services/gateway/apis/http-version3/swagger/definitions/pos.json

```
@@ -521,5 +521,65 @@
 				"type": "object"
 			}
 		}
+	},
+	"PoSRewardsLockedWithEnvelope": {
```

Kindly make the relevant changes wherever necessary apart from the suggested.

Please ensure that going forth we use camel casing. This changes needs to be gradually implemented for all the param/response definitions.
```suggestion
	"posRewardsLockedWithEnvelope": {
```